File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Chts/CHT0113Q00GrdCHT0113Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Chts
{
    [Serializable]
	public class CHT0113Q00GrdCHT0113Info
	{
		private String _disabilityCode;
		private String _disabilityName;
		private String _disabilityKanaName;
		private String _startDate;
		private String _endDate;
		private String _deleteYn;
		private String _pkcht0113;
		private String _n;

		public String DisabilityCode
		{
			get { return this._disabilityCode; }
			set { this._disabilityCode = value; }
		}

		public String DisabilityName
		{
			get { return this._disabilityName; }
			set { this._disabilityName = value; }
		}

		public String DisabilityKanaName
		{
			get { return this._disabilityKanaName; }
			set { this._disabilityKanaName = value; }
		}

		public String StartDate
		{
			get { return this._startDate; }
			set { this._startDate = value; }
		}

		public String EndDate
		{
			get { return this._endDate; }
			set { this._endDate = value; }
		}

		public String DeleteYn
		{
			get { return this._deleteYn; }
			set { this._deleteYn = value; }
		}

		public String Pkcht0113
		{
			get { return this._pkcht0113; }
			set { this._pkcht0113 = value; }
		}

		public String N
		{
			get { return this._n; }
			set { this._n = value; }
		}

		public CHT0113Q00GrdCHT0113Info() { }

		public CHT0113Q00GrdCHT0113Info(String disabilityCode, String disabilityName, String disabilityKanaName, String startDate, String endDate, String deleteYn, String pkcht0113, String n)
		{
			this._disabilityCode = disabilityCode;
			this._disabilityName = disabilityName;
			this._disabilityKanaName = disabilityKanaName;
			this._startDate = startDate;
			this._endDate = endDate;
			this._deleteYn = deleteYn;
			this._pkcht0113 = pkcht0113;
			this._n = n;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/chts/CHT0113Q00GrdCHT0113Info.java
package nta.med.data.model.ihis.chts;

import java.io.Serializable;
import java.util.Date;

public class CHT0113Q00GrdCHT0113Info implements Serializable {
	   private String disabilityCode;
	   private String disabilityName;
	   private String disabilityKanaName;
	   private Date startDate;
	   private Date endDate;
	   private String deleteYn;
	   private Double pkcht0113;
	   private String n;
	public CHT0113Q00GrdCHT0113Info(String disabilityCode,
			String disabilityName, String disabilityKanaName, Date startDate,
			Date endDate, String deleteYn, Double pkcht0113, String n) {
		super();
		this.disabilityCode = disabilityCode;
		this.disabilityName = disabilityName;
		this.disabilityKanaName = disabilityKanaName;
		this.startDate = startDate;
		this.endDate = endDate;
		this.deleteYn = deleteYn;
		this.pkcht0113 = pkcht0113;
		this.n = n;
	}
	public String getDisabilityCode() {
		return disabilityCode;
	}
	public void setDisabilityCode(String disabilityCode) {
		this.disabilityCode = disabilityCode;
	}
	public String getDisabilityName() {
		return disabilityName;
	}
	public void setDisabilityName(String disabilityName) {
		this.disabilityName = disabilityName;
	}
	public String getDisabilityKanaName() {
		return disabilityKanaName;
	}
	public void setDisabilityKanaName(String disabilityKanaName) {
		this.disabilityKanaName = disabilityKanaName;
	}
	public Date getStartDate() {
		return startDate;
	}
	public void setStartDate(Date startDate) {
		this.startDate = startDate;
	}
	public Date getEndDate() {
		return endDate;
	}
	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}
	public String getDeleteYn() {
		return deleteYn;
	}
	public void setDeleteYn(String deleteYn) {
		this.deleteYn = deleteYn;
	}
	public Double getPkcht0113() {
		return pkcht0113;
	}
	public void setPkcht0113(Double pkcht0113) {
		this.pkcht0113 = pkcht0113;
	}
	public String getN() {
		return n;
	}
	public void setN(String n) {
		this.n = n;
	}
	   
}
